src/hotspot/share/opto/subnode.hpp

```diff
@@ -171,6 +171,7 @@ class CmpINode : public CmpNode {
 //------------------------------CmpUNode---------------------------------------
 // Compare 2 unsigned values (integer or pointer), returning condition codes (-1, 0 or 1).
 class CmpUNode : public CmpNode {
+  static const Type* Value_cmpu_and_mask(PhaseValues*, const Node*, const Node*);
```

Comment:
We usually add matching parameter names as found in the source file:
```
  static const Type* Value_cmpu_and_mask(PhaseValues* phase, const Node* in1, const Node* in2);
```
or with the renaming above:
```
  static const Type* Value_cmpu_and_mask(PhaseValues* phase, const Node* andI, const Node* rhs);

```